app/models/work_package.rb

```diff
@@ -73,7 +75,9 @@ class WorkPackage < ActiveRecord::Base
                                                                  User.current) }
   }
 
-  scope :in_status, lambda { |*args| where(status_id: (args.first.respond_to?(:id) ? args.first.id : args.first)) }
+  scope :in_status, -> (*args) do
+                    where(status_id: (args.first.respond_to?(:id) ? args.first.id : args.first))
+                  end
```

Comment:
`end` at 80, 18 is not aligned with `-> (*args) do` at 78, 20 or `scope :in_status, -> (*args) do` at 78, 2